### ISSUE TITLE
syphon filter (USA) black screen

### ISSUE THREAD

**Issue by Reideldm** [notaz/pcsx_rearmed#322]
after the 989 studios movie, black screen right at the beginning of the game, both versions of the game, the standard and the rev1, have been tested

**Comment by saulfabregwiivc** [comment]
Are you using the latest PCSX-ReARMed? @notaz fixed that game recently...

**Comment by Reideldm** [comment]
I am using the e34ef5a updated 2 weeks ago

**Comment by Reideldm** [comment]
I am updating the core from this repo https://github.com/christianhaitian/retroarch-cores/blob/master/arm7hf/pcsx_rearmed_libretro.so.zip because the platform I use is the ps1 classic Do you know of any other site where I can update more frequently?

**Comment by saulfabregwiivc** [comment]
Try asking the current maintainer of PCSX-ReARMed core for PS Classic for see if he/she can update it with the latest changes.

**Comment by notaz** [comment]
I don't know why libretro infra doesn't cover any arm linux systems. Anyway because I called this emu -reARMed I now took my time to setup automatic builds for (almost) all ARM targets that are in libretro makefile:
https://github.com/notaz/pcsx_rearmed/releases/tag/libretro-devbuild
PS Classic should be `pcsx_rearmed_libretro_arm32_classic_armv8_a35.zip` I think.

As for this issue report, it's a duplicate of #317.
